### ISSUE TITLE
when LOAD is pressed it should show that it was clicked

### ISSUE THREAD

**Issue by smurp** [cwrc/HuViz#355]
The user needs reassurance that their click worked. At the moment a long load process leaves the user unclear about whether HuViz is working.

**Comment by smurp** [comment]
Notice the experience on this page:

http://alpha.huviz.dev.nooron.com/#load+https://raw.githubusercontent.com/cwrc/testData/master/sparqlOutputs/unitarian_novelist_biographies.ttl+with+http://sparql.cwrc.ca/ontology/cwrc.ttl

@wolfmaul The fact that loading is proceeding should be conveyed to the user, not just by updating the LOAD button so it shows it has been pressed, but some sort of loading display should be created that works in this case too -- the autoload circumstance.

**Comment by smurp** [comment]
@wolfmaul How about if I find the right place to trigger this `loading progress` screen and you make it, yes?  What are your thoughts about how to display it?  A progress line across the top? A loading overlay thing? A progress cursor at least, I presume?

**Comment by wolfmaul** [comment]
I would be tempted to make some sort of full screen indicator, since the entire interface is 'frozen' until the loading is complete, it is somewhat helpful to make this clear. 

I thought that we had already built such a thing at one point. Wasn't there a loading screen with a progress bar?

**Comment by smurp** [comment]
I've added stub methods if you'd be able to do something attractive in there....

* turn_on_loading_notice()
* turn_off_loading_notice()

I don't remember if there *was* a loading screen.  Could you take a peak for that?

**Comment by wolfmaul** [comment]
Yes, there is a loading screen. I have it sort of working, but it doesn't currently work usefully. It had existed before the reworked unique id names. I have tweaked it so that it now connects, however it doesn't engage usefully when a dataset is loaded, only showing a few moments before the graph is displayed rather than at the beginning of the process (i.e. right after the Load button is clicked). 
The method is called show_state_msg -- I will pull your changes and see if I understand the path to proceed.

**Comment by smurp** [comment]
There is now a **Loading...** dialog turned on by default. 
https://github.com/smurp/huviz/commit/edd53d3f50f0bbbed0176e34762ec1a3beabf5ce

@SusanBrown I think the improved functionality is worth deploying to beta already.  Beautification is coming....

**Comment by SusanBrown** [comment]
This is lovely. Beautification (e.g. placing in center) would be great ;) Very nice too to have the progress indicators with SPARQL lookup.

I think though that the Loading box is not closing when it should when one does a SPARQL look up. I haven't seen it finish yet even with quite small graphs that seem to have loaded completely. e.g. Emecheta, Buchi.

If you save a script and then select it in the command panel, the Loading signal appears before you even press load when it should appear after and it doesn't close either.

**Comment by SusanBrown** [comment]
loading warning disappears too soon sometimes, before one can interact with the graph. Try with the Unitarian novelists dataset. This may be caused or exacerbated by a slow connection.

**Comment by smurp** [comment]
@SusanBrown Now all **file formats** can be loaded and the loading warning is correctly disposed of.  SPARQL next....

https://github.com/smurp/huviz/commit/d5c973b936685535186be3400d361d3f7861f556

**Comment by smurp** [comment]
@SusanBrown The **Loading...** dialog is now correctly disposed of for SPARQL load and reload – on alpha.

https://github.com/smurp/huviz/commit/f4c71dd9fea14fe04b540ea8dc5f82f91f39cbd9
